### PR TITLE
Add square and round app launcher icons

### LINKED ISSUE #212: Add app icon
Change the application icon to use the Human Learning logo instead of having the default droid

### PR DESCRIPTION
**Pull request by Nitwix**
closes #212

### CODE REVIEW

**Comment by Nitwix** [comment]
(for info: I used [this](http://romannurik.github.io/AndroidAssetStudio/icons-launcher.html) wonderful site to make the icon versions)

**Review by wattsecs — APPROVED** [review]
The icon is looking good!

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,8 +11,8 @@
         android:name=".HumanLearningApplication"
         android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
+        android:label="@string/app_name"
```
Comment (bbjubjub2494) [inline app/src/main/AndroidManifest.xml]: that looks like an unnecessary diff, can you remove it?

**Review by bbjubjub2494 — APPROVED** [review]
tested on device, icons show up, LGTM

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Nitwix) [reply app/src/main/AndroidManifest.xml]: you are right. It's reverted now
